media-alpha-dev-us-east-1-policy.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
Should we check this file in?

Reply:
I don't think this is necessary. I'll take it out